src/components/visualizations/chart/highcharts/plugins/renderBubbles.ts

```diff
@@ -0,0 +1,162 @@
+// tslint:disable-line
+/* *
+ * (c) 2010-2019 Torstein Honsi
+ *
+ * License: www.highcharts.com/license
+ *
+ * Modified by Lan Huynh to to support
+ *  - Set default size for bubbles in bubble chart where size value is not provided
+ *  - Fix bubbles is not rendered with min/max config
+ */
+import isNil = require("lodash/isNil");
+export interface IBubbleAxis extends Highcharts.Axis {
+    allowZoomOutside?: boolean;
+    dataMin?: number;
+    dataMax?: number;
+    options?: any;
+    userMin?: number;
+    userMax?: number;
+}
+
+export interface IBubbleSeries extends Highcharts.Series {
+    bubblePadding?: boolean;
+    minPxSize?: number;
+    maxPxSize?: number;
+    zData?: Array<number | null>;
+    radii?: Array<number | null>;
+    getRadii(zMin: number, zMax: number, series: Highcharts.Series): number | null;
+}
+
+export function renderBubbles(Highcharts: any) {
+    const wrap = Highcharts.wrap;
+    const pInt = Highcharts.pInt;
+    const arrayMax = Highcharts.arrayMax;
+    const arrayMin = Highcharts.arrayMin;
+    const pick = Highcharts.pick;
+    const isNumber = Highcharts.isNumber;
+
+    if (Highcharts.seriesTypes.bubble) {
+        // Set default size for bubbles in bubble chart where size value is not provided
+        wrap(Highcharts.seriesTypes.bubble.prototype, "getRadius", function(
+            proceed: any,
+            zMin: number,
+            zMax: number,
+            minSize: number,
+            maxSize: number,
+            value: number,
+        ) {
+            let radius = proceed.apply(this, [zMin, zMax, minSize, maxSize, value]);
+            if (isNaN(value) && isNil(radius)) {
+                // Relative size, a number between 0 and 1 (default is 0.5)
+                // Use Math.sqrt for buble is sized by area
+                radius = Math.ceil(minSize + Math.sqrt(0.5) * (maxSize - minSize)) / 2;
+            }
+            return radius;
+        });
+
+        // #SD-479 fix bubbles is not rendered with min/max config
+        wrap(Highcharts.Axis.prototype, "beforePadding", function(_proceed: any) {
+            const axis: IBubbleAxis = this;
+            const axisLength: number = this.len;
+            const chart: Highcharts.Chart = this.chart;
+            const isXAxis: boolean = this.isXAxis;
+            const min: number = this.min;
+            const range = this.max - min;
+            const activeSeries: IBubbleSeries[] = [];
+
+            let pxMin: number = 0;
+            let pxMax: number = axisLength;
+            let transA = axisLength / range;
+            let zMin = Number.MAX_VALUE;
+            let zMax = -Number.MAX_VALUE;
+
+            function translateSizeToPixel(size: string): number {
+                const smallestSize = Math.min(chart.plotWidth, chart.plotHeight);
+                const isPercent = /%$/.test(size);
```

Comment:
Currently, `seriesOptions.maxSize` and `seriesOptions.minSize` was set default by Highcharts. `number | string` is type of it - exactly, it is number and percentage string. It can be pass from chart configuration be we don't custom it. It not metric value and don't depend on our gdc metric format.

It's work well here. i just need to update type of parameter from `size: string` to `size: string | number`.
Thank You.